### PR TITLE
Add personal page

### PR DESCRIPTION
**Pull request by alexiamcdonald**
This adds my personal page

### CODE REVIEW

File: people/alexiamcdonald.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<<!DOCTYPE html>
+<html>
+<head>
+    <title>Alexia's Page</title>
+</head>
+<body>
+<p> yo I'm awesome! </p>
```
Comment (brendanzab) [inline people/alexiamcdonald.html]: The spaces between the end and beginning tags aren't needed! Also, could you indent the paragraph tag?

File: people/alexiamcdonald.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<<!DOCTYPE html>
```
Comment (brendanzab) [inline people/alexiamcdonald.html]: Extra angle bracket!

File: people/alexiamcdonald.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+<html>
+<head>
+    <title>Alexia's Page</title>
+</head>
+<body>
+<p> yo I'm awesome! </p>
+</body>
+</html>
```
Comment (brendanzab) [inline people/alexiamcdonald.html]: This icon means that you are missing a newline at the end of your file. Git likes you to have new lines at the end! You can set up sublime to automatically add it by adding:

``` json
"ensure_newline_at_eof_on_save": true
```

to your user preferences file.

File: people/alexiamcdonald.html (hunk 1 above)
Comment (alexiamcdonald) [inline people/alexiamcdonald.html]: Sure thing!

File: people/alexiamcdonald.html (hunk 2 above)
Comment (alexiamcdonald) [inline people/alexiamcdonald.html]: Whoops will fix

**Comment by brendanzab** [comment]
Looks good! :+1:

**Pull request merged**